python/dcr_template.py

```diff
@@ -647,28 +641,54 @@ def view_model(self, index):
         @param index  sub-band slice of the DcrModel to extract
         @return Returns a 2D numpy array.
         """
-        model = self.model[index, :, :].copy()
-        weights = self.weights[index, :, :]
-        model[weights > 0] /= weights[weights > 0]
-        model[weights <= 0] = 0.0
+        model = np.zeros_like(self.weights)
+        weight_inds = self.weights > 0
+        model[weight_inds] = self.model[index][weight_inds]/self.weights[weight_inds]
         return model
 
+    def build_dcr_kernel(self, exposures, use_full=None, use_psf=False):
+        """!Calculate the DCR covariance matrix for a set of exposures.
+
+        @param exposures  List of LSST exposures.
+        @param use_full  Flag, set to True to use measured PSF for each exposure,
+                            or False to use the fiducial psf for each.
+        @param use_psf  Flag, set to True to use the PSF for calculating the covariance matrix.
+                            If set to False, then use_full is ignored.
+        """
+        dcr_kernel = []
+        for exp in exposures:
+            visitInfo = exp.getInfo().getVisitInfo()
+            el = visitInfo.getBoresightAzAlt().getLatitude()
```

Comment:
It looks like `getBoresightAzAlt()` just returns a Coord, which does not have `getAzimuth()` or 'getAltitude()'. I'll ask Russell.